### PR TITLE
allow empty file uploads

### PR DESCRIPTION
**Pull request by bernt-matthias**

### CODE REVIEW

**Comment by ic4f** [comment]
If it's only a test so far, maybe convert to draft?

**Comment by nsoranzo** [comment]
Converting to/from Draft restarts all the tests unfortunately, using our `status:WIP` label is faster IMO.

@bernt-matthias Please remember to open PRs from your fork (i.e. not use the web editor to start the PR) to not duplicate the CI tests, thanks!

File: tools/data_source/upload.py [hunk 1]
```diff
@@ -128,8 +128,8 @@ def add_file(dataset, registry, output_path):
     if not os.path.exists(dataset.path):
         raise UploadProblemException('Uploaded temporary file (%s) does not exist.' % dataset.path)
 
-    if check_content and not os.path.getsize(dataset.path) > 0:
-        raise UploadProblemException('The uploaded file (%s) is empty' % dataset.path)
+    # if check_content and not os.path.getsize(dataset.path) > 0:
+    #     raise UploadProblemException('The uploaded file (%s) is empty' % dataset.path)
```
Comment (nsoranzo) [inline tools/data_source/upload.py:133]:
```suggestion
```

**Review by nsoranzo — APPROVED** [review]

**Comment by ic4f** [comment]
> Converting to/from Draft restarts all the tests unfortunately, using our `status:WIP` label is faster IMO.

I didn't know that! Yes, of course, our label is quite sufficient.

**Comment by bernt-matthias** [comment]
Thanks @nsoranzo and excuse my laziness to open a branch from my fork.  Seems also to fix the test problem over here: https://github.com/galaxyproject/galaxy/pull/11063

**Pull request merged**